Componets/Modal/modal.css

```diff
@@ -0,0 +1,28 @@
+.modal-container{
+    background-color: white;
```

Comment:
a variable name can be used, else everything  is fine in code

Reply:
Sure. I will add it. Thanks!